docusaurus/docs/iOS/uikit/getting-started.md

```diff
@@ -80,15 +80,21 @@ ChatClient.shared.connectUser(userInfo: userInfo, tokenProvider: tokenProvider)
 Whenever your users leave the chat component, you should use disconnect to stop receiving chat updates and events while using other features of your app. You disconnect by calling:
 
 ```swift
-chatClient.disconnect()
+chatClient.disconnect {
+    print("disconnect completed")
+}
```

Comment:
Maybe better to show a real example, WDYT?
```suggestion
chatClient.disconnect {
    dismiss(animated: true)
}
```

Reply:
I think it's better to print, since any other action requires some context. Also, `dismiss` is UIKit action, so better to keep it relevant for both UI SDKs. 
If you still think it's better, I can change it, no worries.

Reply:
I think it would be important clarify that they need to wait for the completion block before leaving the screen or going to another screen, it can also be just a comment, like `// dismiss the current screen or go to another screen` 🤔 Otherwise customers might not fully understand what do we mean by "you should wait for the completion block". This is a bit vague, wait for what? They may ask 🤔

Reply:
I like the comment approach more, just updated it 👍